src/event/value.rs

```diff
@@ -202,6 +202,40 @@ impl TryInto<serde_json::Value> for Value {
     }
 }
 
+impl From<remap::Value> for Value {
+    fn from(v: remap::Value) -> Self {
+        use remap::Value::*;
+
+        match v {
+            String(v) => Value::Bytes(v),
```

Comment:
I'm not sure about the naming of `String` here, because really it is still `Bytes`.